README.rst

```diff
@@ -9,70 +9,89 @@ Open Zaak
 
 |build-status| |docs| |coverage| |black| |docker|
 
-API's voor Zaakgericht werken
+Deze repository bevat broncode en documentatie voor productiewaardige API's voor Zaakgericht Werken (ZGW). Deze API's volgen de standaard van VNG Realisatie "API's voor Zaakgericht Werken".
 
-Ontwikkeld door `Maykin Media B.V.`_ in opdracht van Amsterdam, Rotterdam,
-Utrecht, Tilburg, Arnhem, Haarlem, 's-Hertogenbosch, Delft en Hoorn, onder
-regie van `Dimpact`_.
+Zaakgericht Werken
+=========
```

Comment:
In RST moeten de headers (`====`) even lang zijn als de titel zelf, daardoor faalt de docs build nu :)